packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -142,8 +144,8 @@ export const makeInnerVault = (
 
     // Two values from the same moment
     interestSnapshot: manager.getCompoundedInterest(),
-    // @ts-expect-error until https://github.com/Agoric/agoric-sdk/pull/4736
-    runSnapshot: AmountMath.makeEmpty(runBrand),
+    /** @type {any} cast */
```

Comment:
This contract only handles fungible debt, right? Have you tried declaring the `debtBrand` about (around line 120) to `Brand<'nat'>` to see if this cast becomes unnecessary? I wonder if `getIssuerRecord` gives you the assetKind so that you could do a runtime check to be sure.

Reply:
`Brand` is not parameterized yet in master. https://github.com/Agoric/agoric-sdk/pull/4736/ will fix this and the expect-error will error to be cleaned up.